antenna/breakpad_resource.py

```diff
@@ -287,49 +287,44 @@ def extract_payload(self, req):
 
         fs = cgi.FieldStorage(fp=data, environ=req.env, keep_blank_values=1)
 
-        # NOTE(willkg): In the original collector, this returned request
-        # querystring data as well as request body data, but we're not doing
-        # that because the query string just duplicates data in the payload.
-
```

Comment:
This note was here because the old collector both iterated over FieldStorage *and* also added stuff from querystring. However, iterating over FieldStorage picks up both HTTP POST and querystring parameters, so this comment isn't helpful and the old code was probably not right.